ServerCore/Startup.cs

```diff
@@ -87,12 +103,23 @@ public void ConfigureServices(IServiceCollection services)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IHostingEnvironment env)
         {
-            if (env.IsDevelopment())
+            if (env.IsDevelopment() && String.IsNullOrEmpty(Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME")))
+            {
+                app.UseBrowserLink();
+                app.UseDeveloperExceptionPage();
+                PuzzleServerContext.UpdateDatabase(app);
+            }
+            else if (env.IsStaging() && Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME") == "PuzzleServerTestDeploy")
             {
                 app.UseBrowserLink();
                 app.UseDeveloperExceptionPage();
                 PuzzleServerContext.UpdateDatabase(app);
             }
+            else if (env.IsProduction() && Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME") == "Puzzlehunt")
```

Comment:
The capitalization of "Puzzlehunt" is different than the other file in this PR and the comparisons are case sensitive, so at least one of these isn't doing what you want